coderadar-server/coderadar-core/src/main/java/org/wickedsource/coderadar/core/rest/dates/series/DayPoint.java

```diff
@@ -6,24 +6,21 @@
 import com.fasterxml.jackson.databind.annotation.JsonSerialize;
 import java.util.Calendar;
 import java.util.Date;
+import lombok.AllArgsConstructor;
+import lombok.NoArgsConstructor;
 import org.wickedsource.coderadar.core.rest.dates.Day;
 import org.wickedsource.coderadar.core.rest.dates.serialize.DayDeserializer;
 import org.wickedsource.coderadar.core.rest.dates.serialize.DaySerializer;
 
 @JsonTypeName("day")
+@AllArgsConstructor
+@NoArgsConstructor
 public class DayPoint extends Point<Day, Long> {
```

Comment:
same as with `Day`

Reply:
Unfortunately, this doesn't work as @Value makes the fields final, so we can no longer set them using the setters.